### ISSUE TITLE
resume/restart metawrap at some checkpoint(s)

### ISSUE THREAD

**Issue by willnotburn** [bxlab/metaWRAP#54]
My metaWRAP job (with --metabat1) ran out of time on a cluster, but it finished aligning reads and calculating insert gaps.

1. Can I restart metaWRAP and have it skip re-doing the steps it already completed? metaSPAdes has a nifty --restart-from option.

2. What about starting metaWRAP with another binner, like --maxbin2?

If this kind of restart is possible, I could make use of my cluster's SSD-based storage, and run the three binners in parallel, all using the completed alignment files.

**Comment by ursky** [comment]
Thanks for the suggestion. I am currently away until November so I cannot look at it right now. However, feel free to look at the binning.sh file and see if you can comment out the steps that are already done. I can't remember right now but I think it might already continue if the contig abundance files already exists. It's a simple bash script so feel free to hack it. When I return I'll see if I can implement a better option. Thanks for the feedback!

**Comment by ursky** [comment]
I revisited the issue and its looks like the Binning module already automatically continues to binning if the alignment files from previous runs are present. I will add this information in the help message.

**Comment by willnotburn** [comment]
Thanks, Gherman!
